typescript/packages/toolshed/lib/prefixed-logger.ts

```diff
@@ -0,0 +1,57 @@
+export interface Logger {
+  // deno-lint-ignore no-explicit-any
+  info(...args: any[]): void;
+  // deno-lint-ignore no-explicit-any
+  error(...args: any[]): void;
+  // deno-lint-ignore no-explicit-any
+  warn(...args: any[]): void;
+  // deno-lint-ignore no-explicit-any
+  debug(...args: any[]): void;
+}
+
+export class PrefixedLogger implements Logger {
```

Comment:
A little bit of ceremony to automatically add prefixes and collect the logs for reporting. Probably didn't need to home roll but it was easy.